components/BottomNavigation/tests/unit/BottomNavigationItemViewCodingTests.m

```diff
@@ -140,6 +140,32 @@ - (void)testHorizontalMarginLayout {
                              0.001f);
 }
 
+- (void)testAccessibilityIdentifier {
```

Comment:
I think we could move this to BottomNavigationTests.m since these aren't coding tests.  You can use the (Testing) category in that file to avoid having to navigate the view hierarchy as well. ;-)